### DISCUSSION TITLE
[CSS] Target `. . .` 3 dots inside `another-quick-switcher__status__search-command`

### DISCUSSION THREAD

**Discussion by FelipeRearden** [tadashi-aikawa/obsidian-another-quick-switcher#181] [Q&A] [answered]
Hello @tadashi-aikawa !!!!!

Would you mind helping me to target this element inside an AQS modal ?

![6BA33AA9-4616-4367-A04C-C3056543565A](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/85180696/8ab12dfb-9d71-4162-866b-b90fd8ab8fe3)

it's inside `another-quick-switcher__status__search-command`

I want to `display:none` this element because I am using some EMOJI/UNICODE as title for some floating modals to make a minimalistic UI 

it is WIP but would be something like this ....

![A45537B4-BFD6-44DE-A749-80814FEF5FDB](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/85180696/45a30c40-21ee-46b6-9d93-7fb33bbbd822)

Thank you very much for the help @tadashi-aikawa !!!!!

**Comment by tadashi-aikawa** [comment, marked as answer]
@FelipeRearden 
O.K. I'll try it in the next release 👍

**Reply by FelipeRearden** [reply]
Thank you very much @tadashi-aikawa for trying to help me with this CSS !!!!!!!!!!!!!
